engines/director/sprite.cpp

```diff
@@ -65,7 +65,7 @@ Sprite::Sprite(Frame *frame) {
 	_autoPuppet = kAPNone; // Based on Director in a Nutshell, page 15
 	_immediate = false;
 	_backColor = g_director->_wm->_colorWhite;
```

Comment:
I think that `_backColor` should be changed here too. Though, IIRC, that one is used to fill out the Score.

Also, please test this change with Lingo Workshop movies